ammo check/gamedata/configs/text/rus/ui_st_ammocheck.xml

```diff
@@ -1,50 +1,73 @@
 <?xml version="1.0" encoding="windows-1251"?>
 <string_table>
 	<!-- Ammo_Check Messages -->
-	<!-- New strings requiring RU translation.
+	
 	<string id="st_ac_overfull">
 		<text>Overfilled</text>
 	</string>
 	<string id="st_ac_just_one">
 		<text>Just one</text>
 	</string> 
-	-->
+
 	<string id="st_ac_full">
-		<text>Полный магазин</text>
+		<text>������ �������</text>
 	</string>
 	<string id="st_ac_nearly_full">
-		<text>Почти полный</text>
+		<text>����� ������</text>
 	</string>
 	<string id="st_ac_more_half">
-		<text>Больше половины</text>
+		<text>������ ��������</text>
 	</string>
 	<string id="st_ac_about_half">
-		<text>Половина магазина</text>
+		<text>�������� ��������</text>
 	</string>
 	<string id="st_ac_less_half">
-		<text>Меньше половины</text>
+		<text>������ ��������</text>
 	</string>
 	<string id="st_ac_near_empty">
-		<text>Почти пустой магазин</text>
+		<text>����� ������ �������</text>
 	</string>
 	<string id="st_ac_empty">
-		<text>Пустой магазин</text>
+		<text>������ �������</text>
```

Comment:
can you fix these strings?

Reply:
these are actually fixed versions with respect to the game. it's github's encoding support that is broken.